src/nvme/ioctl.c

```diff
@@ -262,16 +262,18 @@ enum nvme_cmd_dword_fields {
 	NVME_VIRT_MGMT_CDW10_RT_MASK				= 0x7,
 	NVME_VIRT_MGMT_CDW10_CNTLID_MASK			= 0xffff,
 	NVME_VIRT_MGMT_CDW11_NR_MASK				= 0xffff,
-	NVME_FORMAT_CDW10_LBAF_SHIFT				= 0,
+	NVME_FORMAT_CDW10_LBAFL_SHIFT				= 0,
 	NVME_FORMAT_CDW10_MSET_SHIFT				= 4,
 	NVME_FORMAT_CDW10_PI_SHIFT				= 5,
 	NVME_FORMAT_CDW10_PIL_SHIFT				= 8,
 	NVME_FORMAT_CDW10_SES_SHIFT				= 9,
-	NVME_FORMAT_CDW10_LBAF_MASK				= 0xf,
```

Comment:
Same here, we need to leave the NVME_FORMAT_CDW10_LBAF_MASK in place. But please update the documentation that it is deprecated.